content/wasm-languages/assemblyscript.md

```diff
@@ -53,32 +53,32 @@ This section provides a basic example of building AssemblyScript from source.
 
 Set up the project like this:
 
-```console
+```bash
 $ npm init
 # Answer questions
-$ npm install --save @assemblyscript/loader
+```
 
-added 1 package, and audited 2 packages in 986ms
+Install the TS compiler:
 
-found 0 vulnerabilities
+```bash
 $ npm install --save-dev assemblyscript
+$ npm install --save @assemblyscript/loader
+```
 
-added 6 packages, and audited 8 packages in 2s
-
-1 package is looking for funding
-  run `npm fund` for details
+Install the WASI shim
 
-found 0 vulnerabilities 
+```bash
+$ npm install --save-dev @assemblyscript/wasi-shim
 ```
 
 Now use `npx` to scaffold out your new project:
 
 ```console
 $ npx asinit .
-Version: 0.19.23
+Version: 0.27.24
 
 This command will make sure that the following files exist in the project
-directory '/Users/technosophos/Code/AssemblyScript/hello':
+directory '/Users/Fermyon/typescript-example':
```

Comment:
Where does this path come from?  It seems unlikely that the user is going to have a `/Users/Fermyon` directory on their filesystem.  And Linux users won't have a `/Users` directory at all.

Reply:
Thank you! The latest commit has these changes in.
I ran it all again locally, works well.